### PR TITLE
A few minor fixes.

### PR DESCRIPTION
**Pull request by DiSkyMaster**
It contains syntax errors, example errors, and typos.

### CODE REVIEW

File: src/main/java/info/itsthesky/disky/elements/properties/DiscordName.java [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
         "You can change name of every entity except member and user by defining a new text.",
         "Check for 'nickname of member' if you want to check / change custom member's name."})
+@Examples("discord name of event-guild")
 public class DiscordName extends SimplePropertyExpression<Object, String> {
 
     static {
         register(
                 DiscordName.class,
                 String.class,
-                "[the] discord name",
```
Comment (SkyCraft78) [inline src/main/java/info/itsthesky/disky/elements/properties/DiscordName.java:22]: Why have you removed the `the` here?

File: src/main/java/info/itsthesky/disky/elements/properties/emotes/EmoteGuild.java [hunk 2]
```diff
@@ -21,7 +21,7 @@ public class EmoteGuild extends SimplePropertyExpression<Emote, Guild> {
         register(
                 EmoteGuild.class,
                 Guild.class,
-                "[the] emo(te|ji) guild",
```
Comment (SkyCraft78) [inline src/main/java/info/itsthesky/disky/elements/properties/emotes/EmoteGuild.java:24]: Same here

File: src/main/java/info/itsthesky/disky/elements/properties/emotes/EmoteURL.java [hunk 3]
```diff
@@ -20,7 +20,7 @@ public class EmoteURL extends SimplePropertyExpression<Emote, String> {
         register(
                 EmoteURL.class,
                 String.class,
-                "[the] emo(te|ji) (ur(i|l)|image [url])",
```
Comment (SkyCraft78) [inline src/main/java/info/itsthesky/disky/elements/properties/emotes/EmoteURL.java:23]: Same here

File: src/main/java/info/itsthesky/disky/elements/properties/messages/MessageChannel.java [hunk 4]
```diff
@@ -11,14 +11,14 @@
...
 @Description("Get the text channel were the message was sent. Can be null if it's in PM or not in guild!")
-@Examples("channel of event-message")
+@Examples("text-channel of event-message")
 public class MessageChannel extends SimplePropertyExpression<Message, TextChannel> {
 
     static {
         register(
                 MessageChannel.class,
                 TextChannel.class,
-                "[discord] [message] [text]( |-)channel",
```
Comment (SkyCraft78) [inline src/main/java/info/itsthesky/disky/elements/properties/messages/MessageChannel.java:21]: the message doesn't especially come from a text channel, but can also come from a news, store, or thread channel.

**Review by SkyCraft78 — CHANGES_REQUESTED** [review]
LGTM overall, just some pattern changes I don't approve of.

File: src/main/java/info/itsthesky/disky/elements/properties/DiscordName.java (hunk 1 above)
Comment (DiSkyMaster) [reply src/main/java/info/itsthesky/disky/elements/properties/DiscordName.java:22]: Because looking at the documents, there are 2 times "the" here.

**Review by SkyCraft78 — APPROVED** [review]

**Pull request merged**